test/runtests.jl

```diff
@@ -82,6 +83,10 @@ function test_match_python_encode(bits_per_char = 2)
 end
 
 Random.seed!(47)
-test_cell_centers_encode_decode()
```

Comment:
Do you no longer need the test?

Reply:
We now call it with different values for `bits_per_char`, though I had sloppily neglected that below (line 89) - fixed.